### DISCUSSION TITLE
Is there any helm chart available for polygon-edge to deploy on kubernetes

### DISCUSSION THREAD

**Discussion by prabhinmp** [0xPolygon/polygon-edge#661] [General]
Is there any helm chart available for polygon-edge to deploy on kubernetes

**Comment by laviniat1996** [comment]
Hi @prabhinmp,
Thank you for your question!

At the moment, there is no helm chart available for Polygon Edge, but we would gladly accept contributions on this side.

**Comment by 0xDones** [comment]
Any updates on this one?
